src/main/java/lk/gov/health/phsp/bean/ClientController.java

```diff
@@ -263,83 +262,69 @@ public void clearExistsValues() {
 
     public void checkPhnExists() {
         phnExists = null;
-        if (selected == null) {
-            return;
-        }
-        if (selected.getPhn() == null) {
-            return;
-        }
-        if (selected.getPhn().trim().equals("")) {
+        if (selected == null || selected.getPhn() == null || selected.getPhn().trim().equals("")) {
             return;
         }
+
         phnExists = checkPhnExists(selected.getPhn(), selected);
     }
 
     public Boolean checkPhnExists(String phn, Client c) {
+        Map param_map = new HashMap();
+
         String jpql = "select count(c) from Client c "
                 + " where c.retired=:ret "
                 + " and c.phn=:phn ";
-        Map m = new HashMap();
-        m.put("ret", false);
-        m.put("phn", phn);
+
+        param_map.put("ret", false);
+        param_map.put("phn", phn);
+
         if (c != null && c.getId() != null) {
             jpql += " and c <> :client";
-            m.put("client", c);
+            param_map.put("client", c);
         }
-        Long count = getFacade().countByJpql(jpql, m);
-        if (count == null || count == 0l) {
-            return false;
-        } else {
-            return true;
-        }
-
+        Long count = getFacade().countByJpql(jpql, param_map);
+        return !(count == null || count == 0l);
     }
 
     public void checkNicExists() {
         nicExists = null;
-        if (selected == null) {
-            return;
-        }
-        if (selected.getPerson() == null) {
-            return;
-        }
-        if (selected.getPerson().getNic() == null) {
-            return;
-        }
-        if (selected.getPerson().getNic().trim().equals("")) {
+        if (selected == null || selected.getPerson() == null
+                || selected.getPerson().getNic() == null
+                || selected.getPerson().getNic().trim().equals("")) {
             return;
         }
         nicExists = checkNicExists(selected.getPerson().getNic(), selected);
     }
 
     public Boolean checkNicExists(String nic, Client c) {
+        Map param_map = new HashMap();
```

Comment:
I think this changing the place of the variable is not useful.

Reply:
My opening is it makes code more readable. That is why I did this whole clean up. Otherwise, I didn't see any issues with the code.

Reply:
The change is fine, I just thought it just thought of effectiveness. We will accept those as it is already done.